docs/development/setup_development.rst

```diff
@@ -35,11 +37,32 @@ be installed via the official documentation links:
 * `Docker CE for Ubuntu`_
 * `Docker CE for Debian`_
 
-Make sure to follow the `Post-installation steps for Linux`_, as well.
+Make sure to follow the `post-installation steps for Linux`_.
+
+
+Fedora and Other Linux Distributions
```

Comment:
"Other Linux Distributions" is a bit broad, unless we want to call out specific Fedora derivatives may be best to just say "Fedora Linux" here.